css-alpha-channel/index.html

```diff
@@ -0,0 +1,79 @@
+---
+feature_name: Alpha Channel Support in CSS Color Parsing
+chrome_version: 52
+feature_id: 5685348285808640
+---
+
+<h3>Background</h3>
+<p>Starting with Chrome 52, CSS rules now accept an alpha (transparency) channel for color values. Do this by specifying a fourth number in contexts that take an RGB value. Alpha channel support takes several forms.</p>
+<ul>
+  <li><code>#RRGGBBAA</code></li>
+  <li><code>#RGBA</code></li>
+  <li><code>rgba(RRR, GGG, BBB, A.A)</code></li>
+</ul>
+<p>Consider the example code below. All three <code>.bottom</code> <code>&lt;div&gt;</code> elements have a red background overlaid with yellow box. <code>.overlay1</code> shows what all the examples would look like without an alpha channel. Yellow overlays red.</p>
+<p>The ramaining <code>&lt;div&gt;</code> elements show what you would see using two methods of specifying an alpha channel. <code>.overlay2</code> shows the new syntax using two-character hex values. The value <code>#ff07</code> would accomplish the same thing.) <code>.overlay3</code> uses the <code>rgba()</code> function. Note that last argument for that function is a decimal value between 0 and 1, not an integer between 0 and 255.</p>
+
+<p>To approximate the alpha channel results in older browsers, the example uses <code>@supports not</code> to set a color that approximates the result of placing yellow over red with a 50% transparency.</p>
```

Comment:
Pretty much every commonly used browser will support `rgba(255, 255, 0, 0.5)` already. It's the hex syntax that's not widely supported.